### PR TITLE
Docs/quickstart improvement

### PR DESCRIPTION
**Pull request by dans77777**
Screenshot (152)
adding screenshot of the formatted links and text

### CODE REVIEW

File: python_airflow_operator.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This class provides integration with Airflow and Armada
 ## armada.operators.armada module
 
 
-### _class_ armada.operators.armada.ArmadaOperator(name, armada_client, job_service_client, queue, job_set_id, job_request_items, \*\*kwargs)
+### _class_ armada.operators.armada.ArmadaOperator(name, armada_client, job_service_client, queue, job_request_items, \*\*kwargs)
```
Comment (jayofdoom) [inline python_airflow_operator.md:15]: This file is automatically generated and should not be manually changed.

File: developer.md [hunk 2]
```diff
@@ -6,10 +6,10 @@ There are many ways you can setup you local environment, this is just a basic qu
 
 ### Pre-requisites
 To follow this section it is assumed you have:
-* Golang >= 1.16 installed [https://golang.org/doc/install](https://golang.org/doc/install)
+* Golang >= 1.18 installed [https://golang.org/doc/install](https://golang.org/doc/install)
```
Comment (jayofdoom) [inline developer.md:9]: Hey, these are good updates, but we do something strange for some of these files. They are copied over from docs/*.md on commit to the master branch. So for the changes to developer.md, quickstart.md, and libraries.md, lets put them in the versions here https://github.com/G-Research/armada/tree/master/docs and automation will pull them over. (So you'll need a new PR, off master branch and targeted at master branch).

**Review by jayofdoom — CHANGES_REQUESTED** [review]
These changes mostly look good, but are just targeted to the wrong place. If you re-target this to master branch, docs/ folder, we'll be able to review it in more detail.

**Comment by dans77777** [comment]
closing the pull request as it has too much mess. will raise new one with all of the changes required

**Pull request closed without merging**